src/include/parser/expression/column_value_expression.h

```diff
@@ -101,8 +127,20 @@ class ColumnValueExpression : public AbstractExpression {
     return expr;
   }
 
+  /**
+   * Copies this ColumnValueExpression with new children
+   * @param children new children
+   * @returns copy of this with new children
+   */
+  std::unique_ptr<AbstractExpression> CopyWithChildren(
+      std::vector<std::unique_ptr<AbstractExpression>> &&children) const override {
+    TERRIER_ASSERT(children.empty(), "ColumnValueExpression should have no children");
+    return Copy();
+  }
+
   common::hash_t Hash() const override {
-    common::hash_t hash = AbstractExpression::Hash();
+    common::hash_t hash = common::HashUtil::Hash(GetExpressionType());
```

Comment:
Remove the hash on `GetExpressionType()` and add back `AbstractExpression::Hash();`

Reply:
Explained in the comment below.